src/components/tags/book-selection.cjsx

```diff
@@ -0,0 +1,39 @@
+React = require 'react'
+_ = require 'underscore'
+
+BOOKS =
```

Comment:
Would it be easier to update constants like these if they were in a file higher up (like in `src/constants.json`)?